src/ImageFlow.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ImageFlow = props => {
+  const { imageAddrs } = props;
+  return (
+    <div>
+      {imageAddrs.map(addr => (
+        <div key={addr}>
+          <img src={addr} alt="" />
+        </div>
+      ))}
+    </div>
+  )
+}
+
+ImageFlow.propTypes = {
+  imageAddrs: PropTypes.arrayOf(
+    PropTypes.string,
+  )
+}
+
+ImageFlow.defaultProps = {
+  imageAddrs: [],
+}
+
+export default ImageFlow;
```

Comment:
The default exports do not export any name, so that the intellisense doesn't work. Every people can rename the modules, so that the refactoring will become a nightmare.